### PR TITLE
Show the effects of all abilities on base power

### PR DESCRIPTION
**Pull request by MacChaeger**
Except the ones dependent on things like turn order. Also not Battery for now.

### CODE REVIEW

**Comment by MacChaeger** [comment]
Let -> var?

**Comment by Zarel** [comment]
This is the client; `let` is not supported by IE9 or any of a number of other common browsers.

File: js/client-battle-tooltips.js [hunk 1]
```diff
@@ -1107,28 +1107,43 @@ var BattleTooltips = (function () {
 		if (!basePower) return basePowerComment;
 
 		// Other ability boosts.
+		var abilityBoosted = 0;
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: You mean `abilityBoost`?

**Comment by Zarel** [comment]
In addition to the build error; do you not have access to `npm test`?

**Comment by MacChaeger** [comment]
I had forgotten to have Travis CI automatically test for my client repository as well; that's fixed now.

**Comment by MacChaeger** [comment]
Oh damn i just realized this is wrong

**Comment by MacChaeger** [comment]
Ok I think it's good now unless you _really_ want Battery supported.

**Comment by Zarel** [comment]
Looks good, thanks!

**Pull request merged**